pos-print/src/main/java/com/clouway/pos/print/adapter/http/CashRegistersService.kt

```diff
@@ -4,17 +4,18 @@ import com.clouway.pos.print.core.CashRegister
 import com.clouway.pos.print.core.ErrorResponse
```

Comment:
this service can be renamed
something like DeviceConfigurationService,DeviceService, DeviceBindingService, DeviceMappingsService 
will fits a lot better.